packages/mui-docs/src/utils/loadScript.ts

```diff
@@ -1,10 +1,7 @@
-export default function loadScript(src: string, position: HTMLElement | null) {
+export default function loadScript(src: string, position: HTMLElement) {
```

Comment:
Code style, no defensive logic in places where it's not their business logic to have.